src/reducers/base-reducer.js

```diff
@@ -73,6 +73,8 @@ const baseReducer = (state = DEFAULT_STATE, action) => {
             return {...state, selectionPlan: null, summit: null, marketingSettings: null, submissionIsClosed: false, allSummitDocs: [], baseLoaded: false};
         case ERROR_RECEIVE_SUMMIT:
             return {...state, selectionPlan: null, summit: null, marketingSettings: null, submissionIsClosed: false};
+        case CLEAR_SELECTION_PLAN:
+            return {...state, selectionPlan: null, submissionIsClosed: false, baseLoaded: false};
         case RECEIVE_SELECTION_PLAN: {
             const entity = {...payload.response};
             return {...state, selectionPlan: entity, submissionIsClosed: false};
```

Comment:
here we do need to check is submission period is valid or not
selectionPlan.submission_begin_date <= now <= selectionPlan.submission_end_date

Reply:
after discussion i will close this one